lib/tasks/build_dependencies.rake

```diff
@@ -56,7 +56,9 @@ namespace :build_dependencies do
     else
       sudo = Process.euid.zero? ? "" : "sudo"
       interactive = $stdin.tty? ? "" : "--non-interactive"
-      cmd = "#{sudo} zypper #{interactive} install #{escaped_list}"
+      # allow package downgrade to avoid failures in CI when the installed
+      # packages are higher than the available ones
+      cmd = "#{sudo} zypper #{interactive} install --allow-downgrade #{escaped_list}"
```

Comment:
Do we really need downgrade? It can cause failures as it will use older packages then it can.

Reply:
I do not like downgrade as well, but the "older" packages are actually not old, they are in the latest published version. At least when running in CI. When running locally user needs to confirm the downgrade manually so this should be pretty safe.

Alternatively we could do this in CI:

```
# sync the installed packages with the OSS repo
zypper dup
# install build dependencies
rake build_dependencies:install
```

The disadvantage is that we would need to touch all places which use `rake build_dependencies:install`. But IIRC that should be just few places...

Reply:
My quick grep found 23 affected packages...

Reply:
ok, so lets allow for now that allow downgrade